binary_transparency/firmware/devices/dummy/rom/rom.go

```diff
@@ -65,13 +65,20 @@ func ResetFromFlags() (Chain, error) {
 		return nil, fmt.Errorf("failed to read transparency bundle: %w", err)
 	}
 
-	// TODO(al): measure firmware and check against manifest expected measurement.
-	// HACK: this should use the firmware measurement field, but it'll do for now.
 	fw, err := ioutil.ReadFile(fwFile)
 	if err != nil {
 		return nil, fmt.Errorf("failed to read firmware: %w", err)
 	}
-	fwHash := sha512.Sum512(fw)
+
+	// measure firmware
```

Comment:
Can this just call out to `dummy.ExpectedMeasurement(fw)`?

Reply:
Yeah, I had some half baked idea about keeping the generation and measurement of this separate, but your suggestion results in simpler code so I'll do that for now and break out later if need be.